Apps/Examples/Examples.xcodeproj/project.pbxproj

```diff
@@ -23,6 +23,8 @@
 		0CC4ECEA25B8AD3000F998B8 /* Custom2DPuckExample.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0CC4ECE925B8AD3000F998B8 /* Custom2DPuckExample.swift */; };
 		0CE8ED88258928200066E56C /* race_car_model.gltf in Resources */ = {isa = PBXBuildFile; fileRef = 0CE8ED87258928200066E56C /* race_car_model.gltf */; };
 		17483DFD2670113300396F8D /* MultiplePointAnnotationsExample.swift in Sources */ = {isa = PBXBuildFile; fileRef = 17483DFC2670113300396F8D /* MultiplePointAnnotationsExample.swift */; };
+		17E28C5C2672A1160033DF0F /* SymbolClusteringExample.swift in Sources */ = {isa = PBXBuildFile; fileRef = 17E28C5B2672A1160033DF0F /* SymbolClusteringExample.swift */; };
+		17E28C602672A4D10033DF0F /* Fire_Hydrants.geojson in Resources */ = {isa = PBXBuildFile; fileRef = 17E28C5F2672A4D00033DF0F /* Fire_Hydrants.geojson */; };
```

Comment:
I don't see `Fire_Hydrants.geojson` included in this PR. Does it need to be added, or was it already in the repo?

Reply:
Oh, strange. Re-added!